launch_testing/test/launch_testing/new_hooks/test_function_scope.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+import launch_testing
+
+import pytest
+
+
+@pytest.fixture(scope='module')
+def order():
+    return []
+
+
+@launch_testing.pytest.fixture
+def launch_description():
+    return launch.LaunchDescription([
+        launch_testing.util.KeepAliveProc(),
+        launch_testing.actions.ReadyToTest(),
+    ])
```

Comment:
@ivanpauno It'd be nice to test a `launch_description` that yields (though I don't see why it wouldn't work)